modules/installing-ove-booting.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-booting_{context}"]
+= Mounting the ISO and booting the machines
+
+To initiate the cluster installation, attach the downloaded ISO to each machine and boot the machines from the ISO.
+
+.Prerequisites
+
+* You have downloaded the installation ISO.
+
+.Procedure
+
+. For each machine in your environment, attach the ISO image to the machine and boot the machine from this image.
+
+. Wait for each machine to boot from the image and display the *Rendezvous node setup* menu.
+
+. Configure the rendezvous node for the cluster installation:
+
+.. On the machine that you want to run as the bootstrap host for the installation process, select *This is the rendezvous node* in the *Rendezvous node setup* menu.
+
+.. In the *Rendezvous node IP selection* menu, select an IP address from the list to use as the rendezvous node IP address and select *Continue*.
```

Comment:
So, the user must take note of the IP selected for the rendenzvous node, as it will be required for the configuration of all the other nodes in the cluster